4.gene_expression_signatures/scripts/1.download-validation-data.py

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Download and Process Neuroblastoma RNAseq Data
+
+# Code from: https://github.com/greenelab/BioBombe/blob/master/10.gene-expression-signatures/0.download-validation-data.ipynb
+# 
+# We are downloading the dataset associated with Harenza et al. 2017. The data profiles RNAseq data from 39 commonly used neuroblastoma (NBL) cell lines.
+# 
+# We are interested in the MYCN amplification status of these cell lines. We will test if the MYCN amplification score learned through the BioBombe signature approach applied to TARGET data generalizes to this cell line dataset.
+# 
+# MYCN Amplification refers to the number of copies of the MYCN gene. MYCN amplification is used as a biomarker for poor prognosis in neuroblastoma patients (Huang and Weiss 2013).
+
+# In[1]:
+
+
+import os
+import requests
+import pandas as pd
+from urllib.request import urlretrieve
+
+from sklearn import preprocessing
+
+
+# In[2]:
+
+
+url = "https://ndownloader.figshare.com/files/14138792"
+name = "2019-01-22-CellLineSTAR-fpkm-2pass_matrix.txt"
+path = os.path.join("data", name)
+
+
+# In[3]:
+
+
+os.makedirs("data", exist_ok=True)
+
+
+# In[4]:
+
+
+urlretrieve(url, path)
+
+
+# In[5]:
+
+
+get_ipython().system(' md5sum "data/2019-01-22-CellLineSTAR-fpkm-2pass_matrix.txt"')
```

Comment:
consider adding comments for this section explaining what is going on

Reply:
done